src/switch_ivr_bridge.c

```diff
@@ -468,12 +469,21 @@ static void *audio_bridge_thread(switch_thread_t *thread, void *obj)
 	//switch_channel_set_flag(switch_core_session_get_channel(session_b), CF_ACCEPT_CNG);
 #endif
 
-	switch_core_session_get_read_impl(session_a, &read_impl);
+	get_read_impl_status = switch_core_session_get_read_impl(session_a, &read_impl);
```

Comment:
since we don't use the var other than checking the result, I think this should be simplify as
```
	if (switch_core_session_get_read_impl(session_a, &read_impl) != SWITCH_STATUS_SUCCESS) {
		switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session_a), SWITCH_LOG_ERROR, "Cannot get read implementation!\n");
		return NULL;
	}
```